Cython/Compiler/Symtab.py

```diff
@@ -158,13 +158,17 @@ class Entry(object):
     # is_fused_specialized boolean Whether this entry of a cdef or def function
     #                              is a specialization
     # is_cgetter       boolean    Is a c-level getter function
+    # known_standard_library_import     Either None (default), False (definitely can't be determined)
```

Comment:
Empty string instead of `False`, maybe? Not sure. It feels wrong to have three different types here. I think an empty string would cover the use cases of being false and not comparing equal to other module names. Are there others?

Reply:
Yeah I think that makes sense